maplibre-winit/src/input/tilt_handler.rs

```diff
@@ -42,6 +42,10 @@ impl TiltHandler {
         } else {
             Deg::zero()
         };
+        // FIXME: If the goal is to maintain the key pressed to tilt, then process_key shouldn't
```

Comment:
Can you add the corresponding issues to these FIXMEs?

After that I think this is ready for merge :)